apps/portal-front/src/components/service/document/one-click-deploy/useOneClickDeployTab.tsx

```diff
@@ -50,10 +51,16 @@ export const useOneClickDeployTab = ({
       });
     }
   };
-
+  const url =
+    documentData.type === ShareableResourceType.OPENAEV_SCENARIO
+      ? `${platformBasePath}/admin/deploy-scenario/${documentData.service_instance?.id}/${documentData.id}`
+      : `${platformBasePath}/dashboard/xtm-hub/${URL_CONFIGS_OPENCTI[documentData.type as keyof typeof URL_CONFIGS_OPENCTI]}/${documentData.service_instance?.id}/${documentData.id}`;
```

Comment:
I find this ternary a bit difficult to read and understand what it does. 
We could put it in a new `computeDeployUrl` function and move computation inside variables.
What do you think ?